android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt

```diff
@@ -148,10 +224,63 @@ class VideoPlaybackService : MediaSessionService() {
         mediaSessionsList.clear()
     }
 
+    override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
+        intent?.let {
+            val playerId = it.getIntExtra("PLAYER_ID", -1)
+            val actionCommand = it.getStringExtra("ACTION")
+
+            if (playerId < 0) {
+                DebugLog.w(TAG, "Received Command without playerId")
+                return super.onStartCommand(intent, flags, startId)
+            }
+
+            if (actionCommand == null) {
+                DebugLog.w(TAG, "Received Command without action command")
+                return super.onStartCommand(intent, flags, startId)
+            }
+
+            val session = mediaSessionsList.values.find { s -> s.player.hashCode() == playerId } ?: return super.onStartCommand(intent, flags, startId)
+
+            handleCommand(commandFromString(actionCommand), session)
+        }
+        return super.onStartCommand(intent, flags, startId)
+    }
+
     companion object {
-        const val COMMAND_SEEK_FORWARD = "SEEK_FORWARD"
-        const val COMMAND_SEEK_BACKWARD = "SEEK_BACKWARD"
+        private const val SEEK_INTERVAL_MS = 10000L
```

Comment:
there no setter to change SEEK_INTERVAL_MS, I think this is creating a regression ...

Reply:
There is no place when we are setting this (and there wasn't earlier either) - now it's private bcs we dont pass it to `VideoPlaybackCallback` as we handle commands int `VideoPlaybackService`
(not sure if I understands it correctly)

Reply:
OK, sorry, I did a mistake, Confusion with: https://thewidlarzgroup.github.io/react-native-video/component/props#controlsstyles